cdap-ui/app/cdap/components/Home/index.js

```diff
@@ -158,6 +159,7 @@ export default class Home extends Component {
             path="/ns/:namespace/profiles/details/:profileId"
             component={ProfileDetailView}
           />
+          <Route exact path="/ns/:namespace/pipelines" component={PipelineList} />
```

Comment:
this was actually a left over while developing. It can be removed. That being said, what do you think of actually removing the entire list view from the angular side, and just keeping it in on the CDAP? the only weird thing is the URL scheme.. where this list view will have `/cdap` instead of `/pipelines`

Reply:
Do it!! 

But do get a word from @bdmogal or @nitinmotgi on the routing url:) We wanted to configure the pipelines to be home in some scenarios (cloud environment). I think this will help achieve that.

Reply:
I think I'll make this change in a subsequent PR as well, to reduce more changes